samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/storage.c

```diff
@@ -114,56 +114,59 @@ void save_on_flash(u8_t id)
 	k_work_submit(&storage_work);
 }
 
-static int ps_set(int argc, char **argv, void *val_ctx)
+static int ps_set(int argc, char **argv, size_t len_rd,
+		  settings_read_cb read_cb, void *cb_arg)
 {
-	int len = 0;
+	size_t len = 0;
 
 	if (argc == 1) {
 		if (!strcmp(argv[0], "rc")) {
-			len = settings_val_read_cb(val_ctx, &reset_counter,
-						   sizeof(reset_counter));
+			len = read_cb(
+			 cb_arg,
+			 &reset_counter,
+			 sizeof(reset_counter));
 		}
 
 		if (!strcmp(argv[0], "gdtt")) {
-			len = settings_val_read_cb(
-				val_ctx,
-				&gen_def_trans_time_srv_user_data.tt,
-				sizeof(gen_def_trans_time_srv_user_data.tt));
+			len = read_cb(
+			 cb_arg,
+			 &gen_def_trans_time_srv_user_data.tt,
+			 sizeof(gen_def_trans_time_srv_user_data.tt));
```

Comment:
Same here, though I realise the old code in this case had the strange style as well.